pkgs/development/python-modules/consul2/default.nix

```diff
@@ -0,0 +1,32 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchPypi,
+  requests,
+  six,
+}:
+
+buildPythonPackage rec {
+  pname = "python-consul2";
+  version = "0.1.5";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "sha256-KchZ3nPhfzarmb6DH9wdkk8W6HciMyd6KO+St7mZlc0=";
+  };
+
+  propagatedBuildInputs = [
+    six
+    requests
+  ];
+
+  # Some tests require Consul binaries.
+  doCheck = false;
```

Comment:
We do have consul packaged in nixpkgs? Adding it to `nativeCheckInputs` should be enough.